common/src/main/java/gov/cms/ab2d/common/service/CoverageService.java

```diff
@@ -45,10 +45,10 @@
 
     /**
      * Retrieve list of coverage periods associated with a contract
-     * @param contract contract to retrieve coverage periods for
+     * @param contractId contract to retrieve coverage periods for
      * @return list of coverage periods associated with a contract
      */
-    List<CoveragePeriod> findAssociatedCoveragePeriods(Contract contract);
+    List<CoveragePeriod> findAssociatedCoveragePeriods(Long contractId);
```

Comment:
Why the change?

Reply:
I think this was based on a merge I had to do.